dev-requirements.txt

```diff
@@ -1,5 +1,4 @@
 ROCmSoftwarePlatform/rocm-recipes
-RadeonOpenCompute/rocm-cmake@04f694df2a8dc9d7e35fa4dee4ba5fa407ec04f8 --build
 -f requirements.txt
 # 1.90+
-danmar/cppcheck@dd05839a7e63ef04afd34711cb3e1e0ef742882f
+danmar/cppcheck@aca3f6fefa6d6b8c5915f15035c7e5908961f6f1
```

Comment:
You can just write `danmar/cppcheck@2.9` if you are using a tagged version. The commit hash is used because it contained some fixes for bugs that were in 1.90.(hence why it says 1.90+).